charts/trustify/templates/helpers/_tracing.tpl

```diff
@@ -7,3 +7,15 @@ Arguments (dict):
 {{ define "trustification.application.tracing.enabled"}}
 {{- .Values.tracing.enabled }}
 {{- end }}
+
```

Comment:
Please add a template documentation section like the others have. Same for all of the new ones.